README.md

```diff
@@ -372,6 +372,16 @@ following customization:
 (setopt clojure-ts-outline-variant 'imenu)
 ```
 
+## Refactoring support
```

Comment:
Might be a good idea to mention at the end of this section all refactoring keybindings in a table and the defcustom for the keymap prefix.